src/sidecar/sidecarManager.ts

```diff
@@ -281,11 +281,11 @@ export class SidecarManager {
   }
 
   /** Called whenever websocket connection goes CONNECTED or DISCONNECTED. */
-  private onWebsocketStateChange(event: WebsocketStateEvent) {
+  private async onWebsocketStateChange(event: WebsocketStateEvent) {
```

Comment:
Only caller is a few lines up: https://github.com/confluentinc/vscode/blob/7faf12425f23114d8fd9af5ce80026efecae4432/src/sidecar/sidecarManager.ts#L276